install/kubernetes/templates/istio-citadel-one-namespace.yaml.tmpl

```diff
@@ -1,35 +1,35 @@
 ################################
-# Istio-CA single namespace
+# Citadel single namespace
 ################################
 # This CA only issues certs/keys for the namespace it runs in.
 # For demo only. Please use istio-cluster-ca.yaml for real-world use cases.
 # Service account CA runs in.
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: istio-ca-service-account
+  name: istio-citadel-service-account
   namespace: {ISTIO_NAMESPACE}
 ---
 # Istio CA
 apiVersion: v1
 kind: Deployment
 apiVersion: extensions/v1beta1
 metadata:
-  name: istio-ca
+  name: istio-citadel
   namespace: {ISTIO_NAMESPACE}
 spec:
   replicas: 1
   template:
     metadata:
       labels:
-        istio: istio-ca
+        istio: citadel
       annotations:
         sidecar.istio.io/inject: "false"
     spec:
-      serviceAccountName: istio-ca-service-account
+      serviceAccountName: istio-citadel-service-account
       containers:
-      - name: istio-ca
-        image: {CA_HUB}/istio-ca:{CA_TAG}
```

Comment:
(Could be a TODO)

We may still have "CA" somewhere. Somewhere using "ca" makes sense like --self-signed-ca, some like here (CA_HUB) may should renamed to CITDADEL_HUB (similar to MIXER_HUB).

Reply:
Yes, good catch. I'll modify them in another PR :)